src/urbit/doc/dev/pokes.md

```diff
@@ -0,0 +1,145 @@
+Toggle the `special-draw-preference` for this game.
+
+Will error if there’s no active game with the specified `game-id`.
+
+### %offer-draw
+```
+[%offer-draw game-id=@dau]
+```
+
+Offer a draw to your opponent in this game.
+
+Will error if there’s no active game with the specified `game-id`.
+
+### %accept-draw
+```
+[%accept-draw game-id=@dau]
+```
+
+Accept a draw offer, archive the game, and kick all subscribers to updates on this game.
+
+Will error if there’s no active game matching the specified `game-id`, or if there’s no draw offer to accept for that game.
+
+### %decline-draw
+```
+[%decline-draw game-id=@dau]
+```
+
+Decline a draw offer.
+
+Will error if there’s no active game matching the specified `game-id`, or if there’s no draw offer to decline for that game.
+
+
+### %claim-special-draw
+```
+[%claim-special-draw game-id=@dau]
+```
+
+Claim a special draw such as the 50-move-rule, archive the game, and kick all subscribers to updates on this game.
+
+Will error if there’s no active game matching the specified `game-id`, if it’s not your turn, or if there’s no special draw available for this game.
+
+
+### %move
+```
+[%move game-id=@dau move=chess-move]
+```
+
+Make a move. If this move ends the game archive it.
+
+Will error if there’s no active game matching the specified `game-id`, if the opponent hasn’t subscribed to this game yet, if it’s not actually your move, or if the move is illegal.
+
+
+### %resign
+```
+[%resign game-id=@dau]
+```
+
+Resign from a game, adding 
+
+Will error if there’s no active game matching the specified `game-id`.
+
+### %request-undo
+```
+[%request-undo game-id=@dau]
+```
+
+Send a request to your opponent to undo the last move, record that you’ve sent an undo request.
+
+Will error if there’s no active game matching the specified `game-id`.
+
+### %decline-undo
+```
+[%decline-undo game-id=@dau]
+```
+
+Decline your opponent’s undo request.
+
+Will error if there’s no active game matching the specified `game-id`, or if there’s no undo request to decline for that game.
+
+### %accept-undo
+```
+[%accept-undo game-id=@dau]
+```
+
+Accept your opponent’s undo request.
+
+## %chess-challenge
+Receive an incoming challenge and put them in the `challenges-received` map.
+
+## %chess-decline-challenge
+Decline an incoming challenge and remove it from the `challenges-received` map.
+
+## %chess-rng
+This poke accepts `chess-rng`s, which are either tagged with `%commit` or `%reveal`.
+
+### %commit
+If you’re sending a challenge, set an `our-num` and `our-hash` and send it to the opponent. Additionally, add it to the `rng-state`  entry for this challenge.
+
+If you’re accepting a challenge, add your `num` and `hash` to the shared `commitment`, then send the completed version back to the challenger.
+
+### %reveal
+If you’re sending a challenge, send the final `commitment` to your opponent.
+
+If you’re accepting a challenge, check that the `commitment` is in order and send your challenger a new game.
```

Comment:
This doesn't actually explain what's happening or why. Also, all of these pokes happen automatically, which begs the question whether we should document them all, given that no user should be manually generating these pokes.